### ISSUE TITLE
Close button (X) not visible in notification when using Android 10 Dark theme

### ISSUE THREAD

**Issue by aibaraiduas** [vanilla-music/vanilla#1002]
- Vanilla Music Version (Settings -> About): 1.0.84
- Android Version: 10
- Phone vendor / model: Google Pixel 3
- Application was downloaded from (use `[x]`):
  - [ ] Play Store
  - [X] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [ ] Yes
  - [ ] No
  - [X] NA / Don't know.

## Description

When Android 10 is set to use the built-in Dark theme, notifications are displayed in black. While Vanilla Music's back/play/forward controls are still visible, the X button used to close the notification is not. It is still there, and it does work if you touch it, but since it is black it is invisible. 

There has been an official dark theme since Android 9, but in Android 10 the notifications were changed to be black. I saw that there was some talk about dark theme notifications, [but only with regards to custom themes](https://github.com/vanilla-music/vanilla/issues/929). This is a report to get the notifications looking correct with the default stock Android dark theme.

## Steps to reproduce the issue

Using stock Android 10, turn on Dark theme (Settings -> Display -> Dark theme).
Play something with Vanilla Music.
Gaze upon the notification.

## Screenshots

Notification with Dark theme enabled:
![Screenshot_20191228-220529](https://user-images.githubusercontent.com/5825551/71552401-a533f600-29ca-11ea-8731-31919ece80cd.png)

Non-dark theme notification for comparison:
![Screenshot_20191228-220550](https://user-images.githubusercontent.com/5825551/71552399-9d745180-29ca-11ea-9c94-28b704085928.png)

**Comment by adrian-bl** [comment]
Turns out we used transparency in the 'X' image.

I changed the image to not use any alpha channels and add a white border around it in commit https://github.com/vanilla-music/vanilla/commit/497af31e52b3f0dd1c9409b1f196fa486215437c - could you check the latest nightly build to see if things are better?

**Comment by aibaraiduas** [comment]
Sorry for not responding sooner! I actually can't test the nightly build: when I download the apk it just gives me the word "error" when I try to install it.

**Comment by adrian-bl** [comment]
Interesting: maybe there was something wrong with the build: could you try again? (the APK should get refreshed every 24 hours)

**Comment by beerisgood** [comment]
@adrian-bl 
Can you maybe push that as beta to F-Droid?

Then I can test it

**Comment by p3732** [comment]
The new icon works, but I looks a bit odd:

![photo_2020-03-09_23-29-58](https://user-images.githubusercontent.com/3109792/76263010-1fe87f80-625e-11ea-91b0-d9bd13934e32.jpg)


Why not implement the same solution as for the other buttons? Or are they defined by the OS?

**Comment by aibaraiduas** [comment]
Yep, same here: the X icon is now visible and works, but it seems to be outlined so that it's darker on the inside. Doesn't seem to match any other icons.
![signal-attachment-2020-04-12-182003](https://user-images.githubusercontent.com/5825551/79081347-9a3b7200-7cea-11ea-84a3-6fac796bb4a8.png)
